EnglishFilter/sections/whitelist_stealth.txt

```diff
@@ -187,7 +187,8 @@
 ! https://github.com/AdguardTeam/AdguardFilters/issues/69616 [Stealth Mode - Referrer]
 @@||link.mazterize.com^$stealth
 ! https://github.com/AdguardTeam/AdguardFilters/issues/69524 [Stealth Mode - Referrer]
-@@||blog.naverblogwidget.com/ExternalWidgetRender.nhn$stealth
+! https://github.com/AdguardTeam/AdguardFilters/issues/85382

```

Comment:
1. Update the old task link.
2. It should be
```
! https://github.com/AdguardTeam/AdguardFilters/issues/85382 [Stealth Mode - Referrer]
```
3. I'd use
```
@@||blog.naverblogwidget.com/ExternalWidgetRender.$stealth
```

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/85394/commits/bf8a63167fd553e2a1fdfecb4bf8ffcc2c50d20e
Is it okay?

Reply:
Please, remove the old comment and it will be ok.
```
! https://github.com/AdguardTeam/AdguardFilters/issues/69524
```

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/85394/commits/33115476226d7afc5c4ff42d7b31a8e4a7f632e9